### PR TITLE
Add post method to VariationPartialView and ElementPartialView

### PR DESCRIPTION
**Pull request by phala**

### CODE REVIEW

File: mote/tests/test_views.py [hunk 1]
```diff
@@ -13,8 +13,25 @@ def setUpTestData(cls):
...
-        url = reverse("mote:element-partial", args=("myproject", "website", "atoms", "button"))
+        url = reverse(
+            "mote:element-partial",
+            args=("myproject", "website", "atoms", "button")
+        )
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
+
+    def test_element_partial_post_request(self):
+        request = self.factory.post(
```
Comment (hedleyroos) [inline mote/tests/test_views.py]: Why not self.client.post ?

Comment (phala) [inline mote/tests/test_views.py]: All I wanted was a POST request object

File: mote/tests/test_views.py [hunk 2]
```diff
@@ -13,8 +13,21 @@ def setUpTestData(cls):
         cls.aspect = models.Aspect("website", cls.project)
         cls.pattern = models.Pattern("atoms", cls.aspect)
         cls.element = models.Element("button", cls.pattern)
+        cls.factory = RequestFactory()
```
Comment (hedleyroos) [inline mote/tests/test_views.py]: You can remove the factory code now.

File: mote/tests/test_views.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 from django.core.urlresolvers import reverse
-from django.test import TestCase
+from django.test import TestCase, RequestFactory
 
-from mote import models
+from mote import models, views
```
Comment (hedleyroos) [inline mote/tests/test_views.py]: You can remove the views import.

**Comment by hedleyroos** [comment]
+1

**Pull request merged**